src/event-schemas/interaction-to-next-paint.json

```diff
@@ -0,0 +1,65 @@
+                },
+                "interactionType": {
+                    "type": "string",
+                    "enum": ["pointer", "keyboard"],
+                    "description": "The type of interaction, based on the event type of the `event` entry that corresponds to the interaction (i.e. the first `event` entry containing an `interactionId` dispatched in a given animation frame). For \"pointerdown\", \"pointerup\", or \"click\" events this will be \"pointer\", and for \"keydown\" or \"keyup\" events this will be \"keyboard\"."
+                },
+                "inputDelay": {
+                    "type": "number",
+                    "description": "The time from when the user interacted with the page until when the browser was first able to start processing event listeners for that interaction. This time captures the delay before event processing can begin due to the main thread being busy with other work."
+                },
+                "processingDuration": {
+                    "type": "number",
+                    "description": "The time from when the first event listener started running in response to the user interaction until when all event listener processing has finished."
+                },
+                "presentationDelay": {
+                    "type": "number",
+                    "description": "The time from when the browser finished processing all event listeners for the user interaction until the next frame is presented on the screen and visible to the user. This time includes work on the main thread (such as `requestAnimationFrame()` callbacks, `ResizeObserver` and `IntersectionObserver` callbacks, and style/layout calculation) as well as off-main-thread work (such as compositor, GPU, and raster work)."
+                },
+                "loadState": {
+                    "type": "string",
+                    "enum": [
+                        "loading",
+                        "dom-interactive",
+                        "dom-content-loaded",
+                        "complete"
+                    ],
+                    "description": "The loading state of the document at the time when the interaction corresponding to INP occurred (see `LoadState` for details). If the interaction occurred while the document was loading and executing script (e.g. usually in the `dom-interactive` phase) it can result in long delays."
+                }
+            },
+            "additionalProperties": false
+        }
+    },
+    "additionalProperties": false,
+    "required": ["version", "value"]
+}
```

Comment:
suggestion: can any of these numbers be negative? if not, we can probably set the `minimum` value to 0.

Reply:
If web vitals is giving bad info, then I think that should be surfaced as is. Otherwise we risk misleading our users because value=0 is typically a perfect score.